templates/docs/examples/patterns/side-navigation/_toggle_script.js

```diff
@@ -24,9 +40,15 @@ function setupSideNavigation(sideNavigation) {
   var toggles = [].slice.call(sideNavigation.querySelectorAll('.js-drawer-toggle'));
 
   toggles.forEach(function (toggle) {
+    var sideNav = document.getElementById(toggle.getAttribute('aria-controls'));
+    var drawerEl = document.querySelector('.p-side-navigation__drawer');
+    var drawerPosition = window.getComputedStyle(drawerEl).position;
+    if (drawerPosition == 'fixed') {
```

Comment:
That seems quite specific and tied to CSS styles, why are we checking this? Is it to detect the initial collapsed state when the site is first loaded on small screen (with navigation off-screen?)